src/test/java/com/amazon/opendistroforelasticsearch/sql/unittest/planner/converter/SQLAggregationParserTest.java

```diff
@@ -55,7 +59,9 @@ public void parseAggWithoutExpressionShouldPass() {
                      "FROM kibana_sample_data_flights " +
                      "GROUP BY dayOfWeek";
         SQLAggregationParser parser = new SQLAggregationParser(new ColumnTypeProvider());
-        parser.parse(mYSqlSelectQueryBlock(sql));
+        MySqlSelectQueryBlock query = mYSqlSelectQueryBlock(sql);
+        List<String> selectMethodNames = prepareRawMethodNames(query, true);
+        parser.parse(query, selectMethodNames);
```

Comment:
Is it possible to make it as the function to avoid duplicate code in all the test cases?

Reply:
Will take another look.